lib/firmware/fsm_msg_cosmos.h

```diff
@@ -0,0 +1,94 @@
+#include "keepkey/firmware/cosmos.h"
+
+void fsm_msgCosmosGetAddress(const CosmosGetAddress *msg)
+{
+    RESP_INIT(CosmosAddress);
+
+    CHECK_INITIALIZED
+
+    CHECK_PIN
+
+    HDNode *node = fsm_getDerivedNode(SECP256K1_NAME, msg->address_n, msg->address_n_count, NULL);
+    if (!node) { return; }
+    hdnode_fill_public_key(node);
+
+    cosmos_getAddress(node->public_key, resp->address);
+
```

Comment:
needs one of these: https://github.com/keepkey/keepkey-firmware/blob/695cbdc8fe04298179936dab0f81bf6a8701814a/lib/firmware/fsm_msg_coin.h#L229